sources/validation-api.asciidoc

```diff
@@ -529,7 +529,7 @@ It is possible to narrow a node instance to its precise type and extract node sp
 [tck-testable]
 --
 `BeanNode`, `PropertyNode` and `ContainerElementNode` host `getContainerClass()` and `getTypeArgumentIndex()`.
-If the node represents an element that is contained in a generic container such as `Optional`, `List` or `Map`,
+If the node represents an element that is contained in a container such as `Optional`, `List` or `Map`,
 the former returns the declared type of the container and the latter returns the index of the affected type argument.
```

Comment:
Why did you remove the `generic` word? We are talking about the type argument so it only makes sense for generic containers.